assets/css/components/buttons.scss

```diff
@@ -1,3 +1,39 @@
+.wp-block-button {
+	.wp-element-button {
+		@extend .govuk-button;
+	}
+
+	&.is-style-start {
+		.wp-element-button {
+			@extend .govuk-button--start;
+
+			&:after {
+				content: url("data:image/svg+xml,%3Csvg class='govuk-button__start-icon' xmlns='http://www.w3.org/2000/svg' width='17.5' height='19' viewBox='0 0 33 40' aria-hidden='true' focusable='false'%3E%3Cpath fill='white' d='M0 0h13l20 20-20 20H0l20-20z' /%3E%3C/svg%3E");
+				margin-left: 10px;
+			}
+
+			&:focus {
```

Comment:
The focus style is kicking in when `:active`, causing the chevron to prematurely change colour.

**suggestion:** append `:not(:active, :hover)`, as per the design system styles (visible in browser dev tools).

Reply:
Ah yes - good spot